aml/org.testeditor.aml.dsl.tests/src/org/testeditor/aml/dsl/tests/parser/validation/AbstractValidationTest.xtend

```diff
@@ -12,8 +12,19 @@
  *******************************************************************************/
 package org.testeditor.aml.dsl.tests.parser.validation
 
+import java.util.UUID
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.EObject
 import org.testeditor.aml.dsl.tests.parser.AbstractParserTest
 
 abstract class AbstractValidationTest extends AbstractParserTest {
 	
-}
\ No newline at end of file
+	protected def <T extends EObject> T register(T model, String fileExtension) {
```

Comment:
The name `register` sounds a bit generic. I would prefer something like `addToResource`